src/includes/constants.php

```diff
@@ -26,8 +26,6 @@
  * - FRAMEWORK_PATH:Absolute filepath, e.g. "/var/www/my-webroot/framework"
  * - PUBLIC_DIR: Webroot path relative to project root - always evaluates to "public"
  * - PUBLIC_PATH: Absolute path to webroot, e.g. "/var/www/project/public"
- * - THIRDPARTY_DIR: Path relative to webroot, e.g. "framework/thirdparty"
- * - THIRDPARTY_PATH: Absolute filepath, e.g. "/var/www/my-webroot/framework/thirdparty"
```

Comment:
That directory doesn't exist in framework anymore, so I'm removing the related constants.
I can't imagine anyone was using them given the only thing in there was the code which takes the template parser `.peg` and builds out the template parser `.php`